test/password.js

```diff
@@ -23,15 +31,15 @@ describe('Password Grant Type ',function() {
             });
     });
 
-    it('POST /token with grant_type="refresh_token" expect same accessToken', function(done) {
+    it('POST /token with grant_type="refresh_token" expect new accessToken always', function(done) {
         request(app)
```

Comment:
@subhashb I have some doubts in this test case. Lets discuss before pulling this pr.